codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsAuthPlugin.java

```diff
@@ -206,19 +206,19 @@ public void writeAdditionalFiles(
         String noTouchNoticePrefix = "// Please do not touch this file. It's generated from template in:\n"
                 + "// https://github.com/aws/aws-sdk-js-v3/blob/main/codegen/smithy-aws-typescript-codegen/"
                 + "src/main/resources/software/amazon/smithy/aws/typescript/codegen/";
-        writerFactory.accept("defaultRoleAssumers.ts", writer -> {
+        writerFactory.accept("src/defaultRoleAssumers.ts", writer -> {
             String resourceName = String.format("%s%s.ts", STS_CLIENT_PREFIX, ROLE_ASSUMERS_FILE);
             String source = IoUtils.readUtf8Resource(getClass(), resourceName);
             writer.write("$L$L", noTouchNoticePrefix, resourceName);
             writer.write("$L", source);
         });
-        writerFactory.accept("defaultStsRoleAssumers.ts", writer -> {
+        writerFactory.accept("src/defaultStsRoleAssumers.ts", writer -> {
             String resourceName = String.format("%s%s.ts", STS_CLIENT_PREFIX, STS_ROLE_ASSUMERS_FILE);
             String source = IoUtils.readUtf8Resource(getClass(), resourceName);
             writer.write("$L$L", noTouchNoticePrefix, resourceName);
             writer.write("$L", source);
         });
-        writerFactory.accept("defaultRoleAssumers.spec.ts", writer -> {
+        writerFactory.accept("test/defaultRoleAssumers.spec.ts", writer -> {
```

Comment:
Create CodegenUtils.TEST_FOLDER which can be reused with procotol_tests
```suggestion
        writerFactory.accept(CodegenUtils.TEST_FOLDER + "/defaultRoleAssumers.spec.ts", writer -> {
```

Reply:
Added a ToDo to be taken up separately https://github.com/awslabs/smithy-typescript/issues/435